javascript/javascript1/week3/script.js

```diff
@@ -0,0 +1,133 @@
+// WARM UP TASK
+const names = [
+  'Peter',
+  'Ahmad',
+  'Yana',
+  'kristina',
+  'Rasmus',
+  'Samuel',
+  'katrine',
+  'Tala',
+];
+
+const nameToRemove = 'Ahmad';
+
+const indexToRemove = names.includes(nameToRemove);
```

Comment:
You are on the right track here! If you replace "Ahmad" with another name in the list, does it still work? 🐛

(Hint: JavaScript is trying to be helpful and converting something to a number for you!)